docs/AddNewVariable.rst

```diff
@@ -20,8 +20,8 @@ NOAA UPP developers who wish to add new variables to the UPP, please follow the
     Hui-Ya Chuang). UPP supports many NOAA operational models and we can not change operational products
     without coordination and advanced notices.
 
-We encourage non NOAA UPP developers to contact the Developmental Testbed Center (DTC) via the UPP
-`forum <https://forums.ufscommunity.org/forum/post-processing>`_ to make them aware of modifications you
+We encourage non NOAA UPP developers to contact the Developmental Testbed Center (DTC) via
```

Comment:
Suggest changing DTC to EPIC here either with this PR or in a near future PR

Reply:
Agreed. It would be good if you could make this change within this PR. Perhaps direct them to Github discussions here too since EPIC is monitoring that space as well?